src/Razor/Microsoft.NET.Sdk.Razor/test/ResolveAllScopedCssAssetsTest.cs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Razor.Tasks;
+using Microsoft.Build.Utilities;
+using Xunit;
+
+namespace Microsoft.NET.Sdk.Razor.Test
+{
+    public class ResolveAllScopedCssAssetsTest
+    {
+
+        [Fact]
+        public void ResolveAllScopedCssAssets_IgnoresRegularCssFiles()
+        {
+            // Arrange
+            var taskInstance = new ResolveAllScopedCssAssets()
+            {
+                StaticWebAssets = new[]
+                {
+                    new TaskItem("TestFiles/Pages/Counter.razor.rz.scp.css", new Dictionary<string,string>{
+                        ["RelativePath"] = "Pages/Counter.razor.rz.scp.css"
+                    }),
+                    new TaskItem("site.css", new Dictionary<string,string>{
+                        ["RelativePath"] = "site.css"
+                    }),
```

Comment:
Nitpick on brace style:

```suggestion
                    new TaskItem("TestFiles/Pages/Counter.razor.rz.scp.css", new Dictionary<string,string>
                    {
                        ["RelativePath"] = "Pages/Counter.razor.rz.scp.css"
                    }),
                    new TaskItem("site.css", new Dictionary<string,string>
                    {
                        ["RelativePath"] = "site.css"
                    }),
```

Reply:
done